app/public/pin.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+	<script src="/jquery/jquery.js">
```

Comment:
I don't like use general libraries like jQuery. They got some much methods inside that we won't be using. And we can all implement those features with vanilla JS or more specific use case libraries (such Axios for only AJAX). And partial can be done by server-side code via template engines like EJS.